composer.json

```
@@ -5,8 +5,8 @@
 	"license": "GPL-2.0-or-later",
 	"require": {
 		"php": ">=7.1",
```

Should we bump the PHP requirement?
```suggestion
		"php": ">=8.0",
```